balancer_pool_states.py

```diff
@@ -142,19 +166,15 @@ def compute_balancer_pool_swaps(ETHERSCAN_API_KEY):
 #      },
 
 
-def compute_balancer_pool_states():
-
-    current_COW = 1475000268143578981182997
-    current_WETH = 227250112810783827603
-    current_time = 1708222403
-
+def compute_balancer_pool_states(CURRENT_TOKEN1 = CURRENT_TOKEN1, CURRENT_TOKEN2 = CURRENT_TOKEN2):
```

Comment:
I am not super firm in python code style but I would have expected something like

```suggestion
def compute_balancer_pool_states(current_toke1 = CURRENT_TOKEN1, current_token2 = CURRENT_TOKEN2):
```
(an argument of a function is not a constant)
Have you checked the code with something like black and pylint?

I would probably remove those arguments altogether.
```python
def compute_balancer_pool_states():
```